core/src/com/unciv/logic/map/mapunit/MapUnit.kt

```diff
@@ -155,6 +155,10 @@ class MapUnit : IsPartOfGameInfoSerialization {
     @Transient
     var cache = MapUnitCache(this)
 
+    /** civ of original owner - relevant for returning captured workers from barbarians */
```

Comment:
Closer to `originalOwner`? Ah okay, has regions... Well that 'Persisted fields' region is already sullied by `class UnitStatus` which simply doesn't belong there... :shrug: would mutual `@see` kdoc help or not?